examples/examples_nodejs_test.go

```diff
@@ -22,23 +22,26 @@ import (
 	"os"
 	"os/exec"
 	"path"
+	"path/filepath"
 	"testing"
 	"time"
 
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
+	"k8s.io/utils/ptr"
```

Comment:
Huh

Reply:
helps passing pointers of constants like `true`